vignettes/UsingSelfControlledCohort.Rmd

```diff
@@ -0,0 +1,222 @@
+---
+title: "Using SelfControlledCohort"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Using SelfControlledCohort}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+
+
+The Self Controlled Cohort method measures the association between an exposure and an outcome by comparing the number of outcomes during an unexposed time at risk to the number of outcomes durning an exposed time at risk. The method is called "Self-Controlled" because each individual in the study contributes person-time to both the exposed and unexposed cohorts. There are seven time points to keep in mind when defining this analysis shown in the diagram below.
+
```

Comment:
I'm not sure how relevant it is, but it's worth discussing that using the SCC package on its is likely to lead to a lot of biased results. It's out of the scope of this Vignette but the EmpiricalCalibration package should ideally be used in conjunction with this, so it might be worth adding a reference.

Reply:
Oh interesting.  That is a good point and I’m not sure it should be out of scope. If it’s something a new user should know then I think it’s appropriate for the vignette. At least we should mention it and point to a longer explanation somewhere else. I actually have a question about this point: does applying empirical calibration change the order of the effects or simply scale all effect estimates by the same amount retaining the order (largest effect is still the largest after calibration).

Reply:
@azimov  - Would you be willing to write a simple example of SCC + MethodEvaluation using Eunomia for the vignette? Any background information or references about why the SCC method is likely to lead to biased results would be really helpful too.